src/index.ts

```diff
@@ -351,7 +339,7 @@ export function main(argString = ""): void {
       myInebriety() <= inebrietyLimit() &&
       (myClass() !== $class`Seal Clubber` || !have($skill`Furious Wallop`))
     )
-      stashItems.push($item`haiku katana`);
+      stashItems.push(...$items`haiku katana, Operation Patriot Shield`);
```

Comment:
In a similar vein of "we don't pull the haiku katana if we're a seal clubber", is it worth not pulling the haiku katana if (we are a TT & we have / can pull the OPS)? and not pulling the OPS if we can pull a haiku katana otherwise? 

I'm not certain how easy that would be to code as I admittedly don't fully understand the withStash inner workings.

Reply:
Will address in subsequent PR